### PR TITLE
Factory pipes to use DECAL_LAYER

### LINKED ISSUE #12711: Kondaru ghostdrone wedge's chair layers under pipes
### Labels

[BUG][trivial][mapping]

### Description

See image

### Steps to reproduce

N/A

### Expected Behavior

Not this

### Additional Information & Screenshots

image

### PR DESCRIPTION
**Pull request by TheTyrant25**
[MAPPING] [TRIVIAL]
## About the PR 
Changes the layer of a ghostdrone decal pipe from 3 to 2.2 (`DECAL_LAYER`), which as the chair is 2.99 layered, fixes the visual bug.
## Why's this needed? 
Fixes #12711

### CODE REVIEW

**Comment by TobleroneSwordfish** [comment]
doesn't this just change one tile of pipe, making it inconsistent with every other tile of pipe?

**Comment by TheTyrant25** [comment]
guess i'll do that then

**Comment by TheTyrant25** [comment]
okay so now all `/obj/decal/fakeobject/pipe` s are on layer 2.9

File: code/obj/decal.dm [hunk 1]
```diff
@@ -366,6 +366,7 @@ obj/decal/fakeobjects/teleport_pad
 	icon = 'icons/obj/atmospherics/pipes/regular_pipe.dmi'
 	icon_state = "intact"
 	anchored = 1
+	layer = 2.9
```
Comment (ZeWaka) [inline code/obj/decal.dm:369]: use defines please

**Review by ZeWaka — CHANGES_REQUESTED** [review]

File: code/obj/decal.dm (hunk 1 above)
Comment (TheTyrant25) [reply code/obj/decal.dm]: i've used `DECAL_LAYER` which is 2.2 (`TURF_LAYER+0.2)

**Pull request merged**